### DISCUSSION TITLE
[Suggestion] Increase render distance

### DISCUSSION THREAD

**Discussion by Xerma** [tsunamayo/Starship-EVO#3978] [Suggestions]
With the saved performance from the removal of voxels and how people are going to, and are, be building bigger, would it be possible to get the render distance increased? 

My main reason for this suggestion is that I may or may not start working on a rather large and spherical design which may or may not be based out of a movie(s) - purely for shits and giggles of course

**Comment by tsunamayo** [comment]
Hi, yes it is planned, although render distance has nothing to do with performance, but more with camera limitations. I need to have two camera, otherwise there is precision issues with the depth buffer.
